LICENSE.GPL.txt

```diff
@@ -0,0 +1,310 @@
+================================================
+ The PyInstaller community hooks licensing terms
+================================================
+
+The PyInstaller community hooks are licensed under the terms of the GNU
```

Comment:
Not only the hooks.

IC. You mean the project name?! Then I suggest to either write `"PyInstaller community hooks"` (in single or double quotes) or `PyInstaller Community Hooks` (capitals, like in a name)

Reply:
Done; See new commits.